storage/storage-mq/src/storage-content/postgresStorageContentRepository.ts

```diff
@@ -1,193 +1,105 @@
 
-    async saveContent(tableIdentifier: string, content: StorageContent): Promise<number> {
-        console.debug(`Saving storage content data`)
-        if (!this.connectionPool) {
-            return Promise.reject("No connnection pool available")
-        }
-
-        delete content.id  // id not under control of client
-
-        // Generate Query-String
-        const data = JSON.stringify(content.data).replace("'", "''") // Escape single quotes
-        const insertStatement = `INSERT INTO "${this.schema}"."${tableIdentifier}" ("data", "pipelineId", "timestamp") VALUES ($1, $2, $3) RETURNING *`
-        const values = [data, parseInt(content.pipelineId), content.timestamp]
-
-        let client!: PoolClient  // Client to execute the query
-        try {
-            client = await this.connectionPool.connect()
-            const { rows } = await client.query(insertStatement, values)
-            console.debug("Content successfully persisted.")
-            return Promise.resolve(rows[0]["id"] as number)
-        } catch (err) {
-            const errMsg = `Could not save content data: ${err}`
-            console.error(errMsg)
-            return Promise.reject(err)
-        } finally {
-            if (client) {
-                client.release()
-            }
-        }
+    const resultSet =
+      await this.postgresRepository.executeQuery(GET_ALL_CONTENT_STATEMENT(POSTGRES_SCHEMA, tableIdentifier), [])
+    const content = this.toContents(resultSet)
+    return Promise.resolve(content)
+  }
+
+  async getContent (tableIdentifier: string, contentId: string): Promise<StorageContent | undefined> {
+    const tableExists = await this.existsTable(tableIdentifier)
+    if (!tableExists) {
+      console.debug(`Table "${tableIdentifier}" does not exist - returning no data`)
+      return Promise.resolve(undefined)
     }
 
-    toContents(resultSet: QueryResult<any>): StorageContent[] {
-      const contents = resultSet.rows as StorageContent[]
-      contents.forEach((x) => {
-        x.id = x.id ? +x.id : undefined // convert to number
-      })
-      return contents
+    const resultSet = await this.postgresRepository.executeQuery(
+      GET_CONTENT_STATEMENT(POSTGRES_SCHEMA, tableIdentifier), [contentId]
+    )
+    const content = this.toContents(resultSet)
+    if (!content || !content[0]) {
+      console.debug(`No content found for table "${tableIdentifier}", id ${contentId}`)
+      return undefined
     }
+    console.debug(
+      `Fetched content for table "${tableIdentifier}", id ${contentId}: ` +
+      `{ pipelineId: ${content[0].pipelineId}, timestamp: ${content[0].timestamp}, data: <omitted in log>}`
+    )
+    return Promise.resolve(content[0])
+  }
+
+  async saveContent (tableIdentifier: string, content: StorageContent): Promise<number> {
+    delete content.id // id not under control of client
+
+    // Generate Query-String
+    const data = JSON.stringify(content.data).replace("'", "''") // Escape single quotes
```

Comment:
Refactor this into a own method which doesn't need a comment next to it to understand the purpose of it.
E.g. `escapeSingleQuotes(input string) : string`